src/content/self-hosted/administration/configuration.mdx

```diff
@@ -517,6 +517,24 @@ The role Okteto assigns to every user. If empty, Okteto will create a default ro
 clusterRole: "role name"
 ```
 
+### user extraRoleBindings
+
+Extra `RoleBindings` to be created for every user's service account. This is useful if you want to grant access to other namespace resources in the cluster.  Disabled by default.
+
+The `RoleBindings` will reference `ClusterRoles` and will be scoped to the namespace specified in the configuration. Here's an example:
+
+```yaml
+user: 
+  extraRoleBindings:
+    enabled: true
+    roleBindings:
+      namespace-name1:
+      - cluster-role1
+      - cluster-role2
+      namespace-name2:
+      - cluster-role3
```

Comment:
If this is a cluster-role, why is the setting called `extraRoleBindings` and not `extraClusterRoles`?

Reply:
This setting is to allow the creation of `RoleBindings` that point to a `ClusterRole`. This is to allow them to get the permissions of the `ClusterRole` but namespaced. 

I get your point about this map not being exactly the rolebindings, since it's a map of `namespace:ClusterRole`. But that wouldn't make it `extraClusterRoles` either. 

This setting is for creating `RoleBindings` so `extraRoleBindings` makes sense, but then the configuration needed is the information we need to create those `RoleBindings`, since we already got the `SA` we only need the `namespace` and `clusterRole`